cvat-core/src/quality-settings.ts

```diff
@@ -182,15 +191,31 @@ export default class QualitySettings {
         const result = await PluginRegistry.apiWrapper.call(this, QualitySettings.prototype.save);
         return result;
     }
+
+    public async create(): Promise<QualitySettings> {
+        const result = await PluginRegistry.apiWrapper.call(this, QualitySettings.prototype.create);
+        return result;
+    }
 }
 
-Object.defineProperties(QualitySettings.prototype.save, {
-    implementation: {
-        writable: false,
-        enumerable: false,
-        value: async function implementation() {
-            const result = await serverProxy.analytics.quality.settings.update(this.id, this.toJSON());
-            return new QualitySettings(result);
-        },
-    },
-});
+Object.defineProperties(
+    QualitySettings.prototype,
+    Object.freeze({
+        save: Object.freeze({
+            writable: false,
+            enumerable: false,
+            value: async function implementation() {
+                const result = await serverProxy.analytics.quality.settings.update(this.id, this.toJSON());
```

Comment:
How is it different from ``cvat.analytics.quality.settings.update.implementation`` declared in ``api-implementation``?

Reply:
Removed extra API